SAMLPortal/Misc/Helpers.cs

```diff
@@ -136,5 +140,61 @@
 			var decodedCert = Convert.FromBase64String(b64Cert);
 			return new X509Certificate2(decodedCert);
 		}
+
+		public static void WriteEnvVariableToFile(string filePath, string key, string value)
+		{
+			if (!key.Contains(" ") && !value.Contains(" "))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'value' before using it.](https://app.codacy.com/manual/zegorax/SAMLPortal/pullRequest?prid=5343548)